app/models/secure_data.rb

```diff
@@ -0,0 +1,28 @@
+# Note: model uses UUID for id which is generated by Postgres. Note that any changes to
+# the way the UUID is generated need to ensure that it generated by a secure random system.
+class SecureData < ApplicationRecord
+  def self.create_and_store(hash)
+    secure_data = new
+    secure_data.store(hash)
+    secure_data.save
```

Comment:
If you are not using the boolean returned by `.save`, I suggest to use `.save!` instead

Reply:
Reasonable point - but to match, I've added a band to the method too `SecureData.create_and_store!`